Source/animation/AnimationProducer.swift

```diff
@@ -88,11 +88,19 @@ class AnimationProducer {
         case .unknown:
             return
         case .affineTransformation:
-            addTransformAnimation(animation, sceneLayer: layer, animationCache: cache, completion: {
-                if let next = animation.next {
-                    self.addAnimation(next)
-                }
-            })
+            do {
+                try addTransformAnimation(animation, sceneLayer: layer, animationCache: cache, completion: {
+                    if let next = animation.next {
+                        self.addAnimation(next)
+                    }
+                })
+            }
+            catch AnimationError.unsupportedAnimation(let errorMessage) {
```

Comment:
No prints. All errors should be thrown when invalid animation added, not produced.